client/src/components/Menu/CategoryDropMenu.js

```diff
@@ -25,9 +25,17 @@ class CategoryDropMenu extends Component {
                 <div className="overlay"></div>
                 {
                     this.state.categoriesList.slice(this.state.categoriesList.length - 5).reverse().map((category) => {
-                        return <div className="category-menu-item" key={category._id}>
-                            <CategoryDropMenuIcon alt={category.icon} src={`/icons/${category.icon}.png`} href={`/categories/${category._id}`} />
-                        </div>
+                         if(category.icon.length>0){
```

Comment:
you can just check for `if(category.icon)`

Reply:
there is icon but the icon value is "" empty

Reply:
that should still be ok with the check `if(category.icon)` - empty is "falsy" value in JavaScript https://developer.mozilla.org/en-US/docs/Glossary/Falsy